src/antares/service/local_services/link_local.py

```diff
@@ -74,15 +75,19 @@ def create_link(
         local_ui = LinkUiLocal.model_validate(ui.model_dump(mode="json", exclude_none=True)) if ui else LinkUiLocal()
 
         properties_ini_file = link_dir / "properties.ini"
-        properties_ini = configparser.ConfigParser()
+        properties_ini = CustomRawConfigParser()
 
         if properties_ini_file.is_file():
             with open(properties_ini_file, "r") as ini_file:
                 properties_ini.read_file(ini_file)
         try:
             properties_ini.add_section(area_to.name)
-        except configparser.DuplicateSectionError as e:
-            raise CustomError(f"Link exists already, section already exists in properties.ini:\n\n{e.message}")
+        except DuplicateSectionError as e:
+            raise LinkCreationError(
+                area_from=area_from.name,
+                area_to=area_to.name,
+                message=f"Link exists already between '{area_from.name}' and '{area_to.name}', section already exists in properties.ini:\n\n{e.message}",
```

Comment:
The first part of the msg is enough I believe: f"Link exists already between '{area_from.name}' and '{area_to.name}'"